4-storage/redshift-storage/sql/atomic-def.sql

```diff
@@ -153,7 +151,40 @@ CREATE TABLE atomic.events (
 	doc_charset varchar(128) encode text255,
 	doc_width integer,
 	doc_height integer,
-	CONSTRAINT event_id_040_pk PRIMARY KEY(event_id)
+
+	-- Currency
+	tr_currency char(3) encode text255,                -- Added in 0.5.0
+	tr_total_base dec(18, 2),                          -- Added in 0.5.0
+	tr_tax_base dec(18, 2),                            -- Added in 0.5.0
+	tr_shipping_base dec(18, 2),                       -- Added in 0.5.0
+	ti_currency char(3) encode text255,                -- Added in 0.5.0
+	ti_price_base dec(18, 2),                          -- Added in 0.5.0
+	base_currency char(3) encode text255,              -- Added in 0.5.0
+
+	-- Geolocation
+	geo_timezone varchar(64) encode text255,           -- Added in 0.5.0
+
+	-- Click ID
+	mkt_clickid varchar(64) encode raw,                -- Added in 0.5.0
+	mkt_network varchar(64) encode text255,            -- Added in 0.5.0
+
+	-- ETL tags
+	etl_tags varchar(500) encode lzo,                  -- Added in 0.5.0
+
+	-- Time event was sent
+	dvce_sent_tstamp timestamp,                        -- Added in 0.5.0
+
+	-- Referer
+	refr_domain_userid varchar(36),                    -- Added in 0.5.0
+	refr_dvce_tstamp timestamp,                        -- Added in 0.5.0
+
+	-- Derived contexts
+	derived_contexts varchar(15000) encode lzo,        -- Added in 0.5.0
+
+	-- Session ID
+	session_id char(36) encode raw,                    -- Added in 0.5.0
+
+	CONSTRAINT event_id_050_pk PRIMARY KEY(event_id)
```

Comment:
Hey @fblundun ! Can we widen some more columns:

https://github.com/snowplow/snowplow.github.com/blob/posts/r63/_posts/2015-03-28-snowplow-r63-red-cheeked-cordon-bleu-released.md#72-updated-fields

Please ping @yalisassoon if any of the updates are unclear.

Many thanks!